### PR TITLE
Created lesson-4, added some text, images, line

### PR DESCRIPTION
**Pull request by IrinaBreiksh**

### CODE REVIEW

**Comment by IrinaBreiksh** [comment]
Added signatures but not sure about sizes

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+p {
+  text-align: center;
+  width: 306px;
+  height: 47px;
+  margin: 120px 567px 20px 567px;
```
Comment (5tronciy) [inline styles/styles.css]: Try to delete the rows 3-5 and you will understand that they have no sense.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
I will send you the result of your work in my browser.
Looks not good.

File: styles/styles.css (hunk 1 above)
Comment (IrinaBreiksh) [reply styles/styles.css]: Ok, I deleted

**Comment by 5tronciy** [comment]
Nothing is changed

**Review by 5tronciy — CHANGES_REQUESTED** [review]
It still differs from the design. Work with Figma deeply and don't be afraid of its complexity.
Check it in browser before next review request pls.

**Comment by IrinaBreiksh** [comment]
> It still differs from the design. Work with Figma deeply and don't be afraid of its complexity. Check it in browser before next review request pls.

Ok,
I rewrote everything

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+hr{
+  width: 100px;
+  height: 2px;
+  background-color: #F5BD41;
+  margin: auto auto;
+}
+
+.advantages{
+  display: inline-block;
```
Comment (5tronciy) [inline styles/styles.css]: What is wrong with the default `display: block;`?

File: index.html [hunk 3]
```diff
@@ -1,41 +1,31 @@
...
-      </p>
-<p>
-    <a href="about/index.html">Об авторе</a>
-</p>
+    <div class="wrapper">
+      <div class="advantages-block">
+        <h1 class="title">What do we offer</h1>
+        <hr class="highlight" />
+        <div class="advantages">
+          <span class="advantages-img"><img class="img" src="./images/support.png" alt="support 24/7"
```
Comment (5tronciy) [inline index.html]: Read about span [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span)

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+.img{
+  width: 100px;
+  height: 100px;
+  padding-bottom: 52px;
+  margin: 90px 126px 56px;
+}
+
+.advantages-text{
+  display: inline;
+  padding-bottom: 120px;
```
Comment (5tronciy) [inline styles/styles.css:34]: I don't know where did you get it. Not from the design.

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.advantages-img {
+  width: 296px;
+  height: 176px;
+}
+
+.img{
+  width: 100px;
+  height: 100px;
```
Comment (5tronciy) [inline styles/styles.css:28]: These sizes are for img wrapper in the design.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
It differs from the design. Use perfect pixel extension to check it.

**Pull request closed without merging**